### DISCUSSION TITLE
Are there seperae sensors for ir and for the reed contact?

### DISCUSSION THREAD

**Discussion by Rico81** [PricelessToolkit/MailBoxGuard#22] [Q&A] [answered]
I've got a mailbox with a small hatch for the postman to deliver the mail and with a bigger door for me to open the mailbox and take out the mail.
What I have in mind is to have delieries checked via the ir sensor and to reset the count when the door gets opened. This would require the mailboxguard to work both mentioned sensors in paralel.
So would that work or is it only one or the other?

Thx
Rico

**Comment by PricelessToolkit** [comment, marked as answer]
yes, basically it is the same (1) trigger mechanism for both.

**Reply by Rico81** [reply]
So to confirm: to check for mail and to reset I'd need two mailboxguards?

**Reply by PricelessToolkit** [reply]
yes, but it will be overkill.  Just Klick reset on the phone!
